### ISSUE TITLE
(on hold) The round button to go to the next/previous page of thumbnails does not work in FF/Safari

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#7129]
**Environment Details**
Spartacus: develop branch
Browser: Firefox & safari
Device: mac
Happens on: http://dev-com-10.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD
Calydon: **disabled**

**Steps to Reproduce**
- Go to a pdp that has multiple thumbnails images
- Click on the "round" button to go to the next/previous page of thumbnail images

**Actual Result**
Nothing happens.
But it works on Chrome.

<img width="999" alt="Screen Shot 2020-04-01 at 10 25 22 PM" src="https://user-images.githubusercontent.com/45409515/78204636-07881100-7468-11ea-8c82-d85310bf0fa9.png">

**Comment by Xymmer** [comment]
To confirm if a styling issue, if not remove styling label and we'll get someone to look at it

**Comment by developpeurweb** [comment]
This is not styling. It could be more related to the use of `tabindex="-1"` which Chrome is ignoring and FF/Safari taking it more strict, therefore Accessibility related. Keyboard navigation can't reach those elements either, in all 3 browsers.

![Screen Shot 2020-04-07 at 9 56 21 AM](https://user-images.githubusercontent.com/15113219/78678205-a40e4100-78b6-11ea-9c04-d9e5183cebf7.png)

![Screen Shot 2020-04-07 at 9 48 25 AM](https://user-images.githubusercontent.com/15113219/78678319-c902b400-78b6-11ea-86f5-4475c2cd3efb.png)

**Comment by tobi-or-not-tobi** [comment]
we have a larger refactor in mind for the carousel where we address the following: 
- accessibility (use the generic keyboard focus directives)
  - control unlocking of the carousel
  - control the slides by keys
  - move focus from one slide to another
- CSS driven size of items (currently it's hardcoded, and the size can't change per item)
- handling touch events
- make use of snappy scrolling
- CSS driven configurations to slide multiple items or a single item

**Comment by giancorderoortiz** [comment]
Still an issue?

**Comment by giancorderoortiz** [comment]
This is still an issue with 3.4. 
Should it be fixed for 4.0?

**Comment by developpeurweb** [comment]
This fix needs to happen on a Major release as there will be many breaking changes related to Carousel refactoring.
Previously proposed changes are in ticket #9481, but they need to be reviewed, as Designers have proposed a new Keyboard Pattern with Accessibility in mind.
